test/package.json

```diff
@@ -1,59 +1,57 @@
 {
-	"name": "wikibase-release-selenium-testing",
-	"version": "0.1.0",
-	"private": true,
+	"name": "test",
+	"version": "3.0.0",
```

Comment:
:)

Reply:
Agreed, but actually the naming and versioning here and in the root are necessitates for nx and npm workspaces to do dependency magic more than something I yet care a lot about. However, I do think what we're being forced to do here will serve us sooner than later in some ways especially around test versions...

Reply:
The smiley above was not meant as making fun of it, more like being amazed how the structure holds for all parts of this repo. I think it is just consistent and makes sense looking forward.